File: pset6/mario/more/mario.py
from cs50  import get_int

while True:
    height = get_int("Enter height: ")

    if height in range(1,9):
        try:
            for row in range (1, height +1):
                print(" " * (height - row) + "#" * row + " " * 2 + "#" * row)

            break
        except ValueError:
            print("Input must be between 1 and 8")

File: pset1/mario/less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {

   height = get_int ("Height:");

    }
    while (height < 1 || height > 8);

    // outer loop iterating through each row
     for (int i= 1; i <= height; i++)
    {
        //loop for spaces
        for (int h=i; h < height; h++)
        {
            printf(" ");
        }

        // innner loop to print hashes
        for(int j=1; j <= i; j++)
        {
            printf("#");
        }

        printf("\n");
    }


}

